tutorials/driving.md

```diff
@@ -18,43 +18,19 @@ The simplest way to get your robot driving is to use a keyboard application on y
 In order to control the Turtlebot 4 while running the application on your pc, you must set up the [networking](../setup/networking.md) between your robot and your pc, so that they can communicate over ros topics.
 ```
 
-{% tabs keyboard %}
-{% tab keyboard galactic %}
-
 You can install the `teleop_twist_keyboard` package on your PC by running the following commands:
 
 ```bash
 sudo apt update
-sudo apt install ros-galactic-teleop-twist-keyboard
+sudo apt install ros-${ROS_DISTRO}-teleop-twist-keyboard
```

Comment:
Same thought, can we add a note above this to make sure they source their ros environment first? We generally can't take that for granted with the beginner users

Reply:
As previously, reverted to tabs and no envar